lib/Deprecated.js

```diff
@@ -564,8 +564,9 @@ class iConn {
       // pre v1.0 falls back to idb transport when host is not specified
       if (option.host) {
         options.transport = 'rest';
+        // construct the url host:port/path
+        options.transportOptions.url = `http://${option.host}:${option.port || 80}${option.path || '/'}`;
       }
-      options.transportOptions = { ...options.transportOptions, ...option };
```

Comment:
Looks like this change is the cause of [unit test failure](https://github.com/IBM/nodejs-itoolkit/pull/272/checks?check_run_id=630828052#step:6:273). We should still append user passed options to the transportOptions like we did before.

Reply:
Oops. I should have caught that.